scripts/l10n.ts

```diff
@@ -284,6 +287,63 @@ async function generateReport(missingTranslations: MissingTranslations) {
   return lines.join('\n');
 }
 
+type DocumentationReportItem = {
+  languageTag: string;
+  importName: string;
+  localeName: string;
+  missingKeysNb: number;
+  totalKeysNb: number;
+  githubLink: string;
+};
+const generateDocReport = async (
+  missingTranslations: MissingTranslations,
+  baseTranslationsNumber,
+) => {
+  const workspaceRoot = path.resolve(__dirname, '../');
+
+  packagesWithL10n.forEach(async ({ key: packageKey, documentationReportPath }) => {
+    const documentationReport: DocumentationReportItem[] = [];
+    Object.entries(missingTranslations).forEach(([importName, infoPerPackage]) => {
+      const info = infoPerPackage[packageKey];
+      if (info == null) {
+        return;
+      }
+      const githubLink = `${SOURCE_CODE_REPO}/blob/master/${info.path}/`;
+
+      const languageTag = `${importName.slice(0, 2).toLowerCase()}-${importName
+        .slice(2)
+        .toUpperCase()}`;
+      const localeName = localeNames[languageTag];
+
+      if (localeName === undefined) {
+        throw new Error(
+          [
+            `locale tage ${languageTag} is not associated to a locale name.`,
+            'If this tag is correct, add its name to the file `scripts/localeNames.js`',
+          ].join('\n'),
+        );
+      }
+      documentationReport.push({
+        languageTag,
+        importName,
+        localeName,
+        missingKeysNb: infoPerPackage[packageKey].missingKeys.length,
+        totalKeysNb: baseTranslationsNumber[packageKey],
```

Comment:
Nitpick, but usually `number` is shortened to `no`, isn't it?
Or maybe we could ho with `Count` instead? 🤔 
```suggestion
        missingKeysCount: infoPerPackage[packageKey].missingKeys.length,
        totalKeysCount: baseTranslationsNumber[packageKey],
```

Reply:
Indeed, I'm using French abbreviations hoping they are the same since "nombre" (FR) is close to "number" (EN)

But of course, they use an "o" 😄